test/E2ETest/TyeRunTests.cs

```diff
@@ -900,6 +904,130 @@ public async Task DockerFileChangeContextTest()
+        {
+            using var projectDirectory = CopyTestProjectDirectory("multi-targetframeworks");
+
+            var projectFile = new FileInfo(Path.Combine(projectDirectory.DirectoryPath, "tye-no-buildproperties.yaml"));
+            var outputContext = new OutputContext(_sink, Verbosity.Debug);
+            var application = await ApplicationFactory.CreateAsync(outputContext, projectFile, "netcoreapp3.1");
+
+            var handler = new HttpClientHandler
+            {
+                ServerCertificateCustomValidationCallback = (a, b, c, d) => true,
+                AllowAutoRedirect = false
+            };
+
+            var client = new HttpClient(new RetryHandler(handler));
+
+            await RunHostingApplication(application, new HostOptions(), async (app, uri) =>
+            {
+                // make sure it is running
+                var backendUri = await GetServiceUrl(client, uri, "multi-targetframeworks");
+
+                var backendResponse = await client.GetAsync(backendUri);
+                Assert.True(backendResponse.IsSuccessStatusCode);
+
+                var responseContent = await backendResponse.Content.ReadAsStringAsync();
+                Assert.Contains(".NET Core 3.1", responseContent);
+            });
+        }
+
+        [ConditionalFact]
+        [SkipIfDockerNotRunning]
+        public async Task RunCliArgOverrideYamlMultipleTargetFrameworksTest()
+        {
+            using var projectDirectory = CopyTestProjectDirectory("multi-targetframeworks");
+
+            var projectFile = new FileInfo(Path.Combine(projectDirectory.DirectoryPath, "tye-with-netcoreapp21.yaml"));
+            var outputContext = new OutputContext(_sink, Verbosity.Debug);
+            var application = await ApplicationFactory.CreateAsync(outputContext, projectFile, "netcoreapp3.1");
+
+            var handler = new HttpClientHandler
+            {
+                ServerCertificateCustomValidationCallback = (a, b, c, d) => true,
+                AllowAutoRedirect = false
+            };
+
+            var client = new HttpClient(new RetryHandler(handler));
+
+            await RunHostingApplication(application, new HostOptions(), async (app, uri) =>
+            {
+                // make sure it is running
+                var backendUri = await GetServiceUrl(client, uri, "multi-targetframeworks");
+
+                var backendResponse = await client.GetAsync(backendUri);
+                Assert.True(backendResponse.IsSuccessStatusCode);
+
+                var responseContent = await backendResponse.Content.ReadAsStringAsync();
+                Assert.Contains(".NET Core 3.1", responseContent);
+            });
+        }
+
+        [ConditionalFact]
+        [SkipIfDockerNotRunning]
+        public async Task RunIgnorePluralTargetFrameworksWhenSingularFormExistsTest()
+        {
+            using var projectDirectory = CopyTestProjectDirectory("multi-targetframeworks-and-targetframework-too");
```

Comment:
This scenario isn't valid, let's remove the test and the test project.

Reply:
Yeah I remember we spoke about that.
Do you want me to remove it or di you want to do it ?